src/profile_manager/views.py

```diff
@@ -149,6 +156,71 @@ def dispatch(self, *args, **kwargs):
         else:
             raise Http404("Sorry, this profile isn't yours!")
 
+    def post(self, request, *args, **kwargs):
+        forms = self.get_forms()
+
+        # check if all form instances are valid else ...
+        if all([form.is_valid() for form in forms]):
+            return self.form_valid(forms)
+        return self.form_invalid(forms)
+
+    def get_context_data(self, **kwargs):
+        profile = self.get_object()
+        context = {}
```

Comment:
why no call to super? `context = super().get_context_data(**kwargs)` is there a reason you are resetting the context to empty = {} ?

Reply:
```super().get_context_data(**kwargs)``` starts throwing throwing errors if i dont include a bunch of unnecessary variables. Calling super is not unnecessary anyway since all of the extra kwargs added is not used